CHANGELOG.md

```diff
@@ -0,0 +1,55 @@
+# Changelog
```

Comment:
I'd add a note about semantic versioning. And adding categories to the messages *might* be useful :)